src/fiat-connect-client.ts

```diff
@@ -106,58 +88,8 @@ export class FiatConnectClientImpl implements FiatConnectApiClient {
           issuedAt = new Date()
         }
       }
-      const expirationTime = new Date(issuedAt.getTime() + SESSION_DURATION_MS)
-      const siweMessage = new SiweMessage({
-        domain: new URL(this.config.baseUrl).hostname,
-        // Some SIWE validators compare this against the checksummed signing address,
-        // and thus will always fail if this address is not checksummed. This coerces
-        // non-checksummed addresses to be checksummed.
-        address: ethers.utils.getAddress(this.config.accountAddress),
-        statement: 'Sign in with Ethereum',
-        uri: `${this.config.baseUrl}/auth/login`,
-        version: '1',
-        chainId: NETWORK_CHAIN_IDS[this.config.network],
-        nonce: generateNonce(),
-        issuedAt: issuedAt.toISOString(),
-        expirationTime: expirationTime.toISOString(),
-      })
-      const message = siweMessage.prepareMessage()
-      const body: AuthRequestBody = {
-        message,
-        signature: await this.signingFunction(message),
-      }
+      await this._siweClient.login({ issuedAt, headers: this._getAuthHeader() })
```

Comment:
Can we move the clock coordination/sync stuff above into the SIWE client?

Reply:
oops completely missed this, should be fixed now.